src/mpifx_init.F90

```diff
@@ -6,7 +6,7 @@ module mpifx_init_module
   implicit none
   private
 
-  public :: mpifx_init
+  public :: mpifx_init, mpifx_init_thread
```

Comment:
mpifx_init is equivalent to mpifx_init_thread at the MPI_THREAD_SINGLE level. Is it worth either noting this somewhere, or getting the mpi_init call to redirect to mpifx_init_thread internally?

Reply:
Well, initially I had one function with optional arguments. However, since on the mpi-level those are two distinct functions, I decided to make them distinct in mpifx as well to be as close to mpi as possible.

Reply:
OK, but might still be worth a comment, perhaps in the mpifx_init, if only to raise awareness of the correct thread call.